frontend/translations/en.po

```diff
@@ -6987,3 +6987,9 @@ msgstr "Autosaved versions will be kept for %s days."
 #, unused
 msgid "workspace.viewport.click-to-close-path"
 msgstr "Click to close the path"
+
+msgid "dashboard.mark-all-as-read.success"
+msgstr "Marked all notifications as read"
+
+msgid "labels.read"
```

Comment:
The translation strings under `label` namespace should be a bit more literal, because the represent the exact message. So I suggest using `label.mark-all-as-read` for this case